simplivity-swagger.json

```diff
@@ -12,7 +12,7 @@
         }
     },
     "host": "localhost",
-    "basePath": "",
+    "basePath": "/",
```

Comment:
I think you need the basePath to be empty to get the UI to work correctly.

Reply:
okay! but this was in the api-docs.json for psi21 artifactory